### PR TITLE
added verification to createhtml

### PR DESCRIPTION
**Pull request by KHardt**
## Link to Ticket
Closes # https://github.com/NSS-Therapeutic-Raccoons/WorkforceManagement/issues/1

## Description of Proposed Changes

- Modified Department test to check createhtml for input

## Steps to Test

git fetch --all
git checkout KH-Department-Test
run the tests and verify in browser

## Impacted Areas in Application

Department Test

## Mentions @username

@khardt
@figamus
@jeremiah3643
@ricky-bruner
@askingalot

## Definition of Done

1. The project must be fully documented. This includes the following:
    1. Complete README that documents the steps to install the code, how to install any dependencies, or system configuration needed.
    1. Every class must be documented with purpose, author, and methods.
    1. Every method must be documented with purpose and argument list - which itself must contain a short purpose for each argument.
1. The project must be able to run fully, and without errors, on each teammate's system.
1. Fulfills every requirement.
Every line of code has been peer reviewed.
1. For projects that require unit testing, core functionality must be identified and have at least one test for each.

### CODE REVIEW

**Comment by ricky-bruner** [comment]
1. Please pull new master and fold them into this branch. It does not have the last two masters on it.
2. Please Comment your code blocks with Andy's comment guidelines.

**Review by ricky-bruner — APPROVED** [review]
Your comments are a bit weak, but everything is working and functioning properly

File: BangazonWorkforce/BangazonWorkforce.IntegrationTests/Config.cs [hunk 1]
```diff
@@ -0,0 +1,13 @@
+﻿namespace BangazonWorkforce.IntegrationTests
```
Comment (askingalot) [inline BangazonWorkforce/BangazonWorkforce.IntegrationTests/Config.cs]: This file shouldn't be included

**Review by askingalot — CHANGES_REQUESTED** [review]
The link in the PR description doesn't work

**Review by askingalot — APPROVED** [review]

**Pull request merged**